cmd_args.py

```diff
@@ -79,6 +79,14 @@ def parse_args():
     parser.add_argument('--infer_files', type=str, default="",
                         help='list of data files in the form of [file1, file2, ...];'
                         'or a single path to a folder of files')
+    parser.add_argument('--intervals_file', type=str, default="",
+                        help='bed file containing the chr and interval values for inference')
+    parser.add_argument('--sizes_file', type=str, default="",
+                        help='bed file containing the chr and max size values')
+    parser.add_argument('--round', type=int,
+                        help='round the output to this value')
+    parser.add_argument('--infer_format', type=str, default="bedgraph",
+                        help='save the inference output to bigiwig or bedgraph.')
```

Comment:
let's update this to say `additionally save the inference output to bigwig` and rename the option to `gen_bigwig` for now, till we exclusively generate one of the other

Reply:
done.